float32 time
float32 position
float32 velocity
float32 acceleration